Makefile

```diff
@@ -12,7 +12,7 @@ ifeq ($(SGX_MODE), HW)
 else ifeq ($(SGX_MODE), SW)
 	ext := sw
 else
-	$(error SGX_MODE must be either HW or SW)
+$(error SGX_MODE must be either HW or SW)
```

Comment:
why do you remove the formatting? this makes the code harder to read

Reply:
because this is required by the Makefile format to work correctly. try running `make SGX_MODE=foo` with and without this indentation. Without the indentation, this is interpreted as an error invocation. With the indentation it's interpreted as a recipe step without a declaration.